package.json

```
@@ -16,8 +16,8 @@
     "@essential-projects/iam_contracts": "^3.1.0",
     "@essential-projects/errors_ts": "^1.2.0",
     "@process-engine/solutionexplorer.contracts": "1.0.0",
-    "@process-engine/solutionexplorer.repository.contracts": "4.0.0",
-    "@process-engine/solutionexplorer.service.contracts": "4.0.0"
+    "@process-engine/solutionexplorer.repository.contracts": "feature~add_optional_path",
+    "@process-engine/solutionexplorer.service.contracts": "feature~add_optional_path"
   },
   "devDependencies": {
     "@essential-projects/tslint-config": "1.0.0",
```

Might want to think about changing this to

```suggestion
    "@essential-projects/tslint-config": "^1.0.0",
```

To get the most current version of this.